### PR TITLE
Yuchaodev

### PR DESCRIPTION
**Pull request by Leezki**
### Why:
* Explain why you are creating this PR - delete this line before finalizing the PR.

### What:
* What does this PR accomplish? What changed, etc.. - delete this line before finalizing the PR.

### Checklist
- [ ] Finished My Changes
- [ ] Automated Unit Tests

### CODE REVIEW

File: ec2setup/algorithms/SmartCharging/postgres_info.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"DB_HOST": "script-postgresql-db.cfnv951vt1nj.us-east-1.rds.amazonaws.com", "CLEANED_DATA_BUCKET_NAME": "script.test.lambda.clean", "POSTGRES_USER": "script_admin", "POSTGRES_PASSWORD": "script_passwd", "POSTGRES_DB": "scriptdb"}
```
Comment (hooshooo) [inline ec2setup/algorithms/SmartCharging/postgres_info.json:1]: don't hardcode it

File: run.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        print(stdout.read())
+        print('error message:')
+        print(stderr.read())
+        # close the client connection once the job is done
+        client.close()
+
+    except Exception as e:
+        print(e)
+
+run_algorithm()
```
Comment (hooshooo) [inline run.py:125]: format

File: variables.env [hunk 3]
```diff
@@ -1,3 +1,4 @@
 POSTGRES_USER=script_admin
 POSTGRES_PASSWORD=script_passwd
 POSTGRES_DB=scriptdb
+CLEANED_DATA_BUCKET_NAME=script.test.lambda.clean
```
Comment (hooshooo) [inline variables.env:4]: don't hardcode it

**Review by wyq730 — COMMENTED** [review]
Nice, I think Yuchao (@Leezki) can merge as long as there is no conflict.

**Pull request closed without merging**